packages/scxa-tsne-plot/src/ClusterTSnePlot.js

```diff
@@ -37,7 +37,7 @@ const tooltipHeader = (clusterType, series, point) => {
 const ClusterTSnePlot = (props) => {
   const {ks, perplexities, metadata, selectedPerplexity, onChangePerplexity, selectedColourBy, onChangeColourBy, clusterType} = props  // Select
   const {plotData, highlightClusters, height, tooltipContent} = props   // Chart
-  const {loading, resourcesUrl, errorMessage, showControls} = props   // Overlay
+  const {loading, resourcesUrl, errorMessage, showControls, initialCellTypeValues} = props   // Overlay
```

Comment:
My initial implementation was using `selectedColorBy` in that case we don't need any new props. After discussion with @alfonsomunozpomer regarding that way of implementation, we decided that it would be better to introduce `props` for `cellTypes` because it would give us a bit more flexibility in the case when `cellTypes` are changed in the future and then we can just pass the new ones as props from the `jsp` etc. In the initial case, we would then have to change them in component again as they would be hard coded here.